fido/fido.py

```diff
@@ -335,6 +342,9 @@ def fetch(
     :param connect_timeout: maximum time allowed to establish a connection
         in seconds.
     :param tcp_nodelay: flag to enable tcp_nodelay for request
+    :param gzip_enabled: flag to enable decompressing gzipped responses.
```

Comment:
Nit: Some concern with the wording, if I'm a developer considering how to use this, I would not know from reading this what to expect. How do I get a gzipped response? 

I think it'd be more clear to provide more technical detail, just mention the exact things we do when this is True (set request header, decompress if it was sent compressed).